src/main/java/com/leverx/learningmanagementsystem/btp/registry/controller/RegistryController.java

```diff
@@ -0,0 +1,62 @@
+package com.leverx.learningmanagementsystem.btp.registry.controller;
+
+import com.leverx.learningmanagementsystem.btp.registry.model.RegistryRequestDto;
+import com.leverx.learningmanagementsystem.btp.registry.service.TenantRegistryService;
+import lombok.AllArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.context.annotation.Profile;
+import org.springframework.security.oauth2.server.resource.authentication.JwtAuthenticationToken;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.ResponseStatus;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.Map;
+import java.util.stream.Collectors;
+
+import static org.springframework.http.HttpStatus.NO_CONTENT;
+import static org.springframework.http.HttpStatus.OK;
+
+@RestController
+@RequestMapping("/api/v1")
+@Profile({"cloud", "local"})
+@Slf4j
+@AllArgsConstructor
+public class RegistryController {
```

Comment:
better to name `SubscriptionController` to narrow down scope of responsibility of this controller. Registry is too abstract